app-android/src/main/res/xml/locales_config.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<locale-config xmlns:android="http://schemas.android.com/apk/res/android">
+    <locale android:name="en"/>
+    <locale android:name="zh"/>
```

Comment:
This app currently provides only simplified Chinese translations.
I think it should be separated simplified Chinese and traditional Chinese.
Could you please set name `zh-CN` ? 🙏 

```suggestion
    <locale android:name="zh-CN"/>
```

Reply:
OK. I've modified. Thank you.